pkg/install/resources.go

```diff
@@ -0,0 +1,161 @@
+package install
+
+import (
+	"fmt"
+
+	appsv1beta1 "k8s.io/api/apps/v1beta1"
+	corev1 "k8s.io/api/core/v1"
+	rbacv1beta1 "k8s.io/api/rbac/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type Option func(*config)
```

Comment:
I don't think this needs to be exported?

Can you move all the option-related bits closer to where they're used (`Deployment()`). Also maybe we should call it `deploymentOption` instead of just `option`?